espanol/bombas/bombas.md

```diff
@@ -0,0 +1,7 @@
+Grandes bombas caían sobre todo Canudos. Hombres, mujeres, niños y ancianos; todos eran consumidos por las explosiones y el fuego, un fuego tan intenso que no dejaba nada a su paso.
+
+En medio del caos, el Consejero tuvo una gran idea:
+
+[Sacó una guitarra y empezó a bailar](bailar/bailar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.